mtz-update-password.html

```diff
@@ -76,6 +129,10 @@
         autocomplete="[[autocomplete]]"
         invalid="[[__passwordInvalid]]"
       ></paper-input>
+      <template is="dom-if" if="[[!hideStrengthMeter]]">
```

Comment:
This is forced on which will break all apps that depend on it, make it opt-in instead

Reply:
Yeah I was thinking about it, I’ll just change it to showProgressMeter instead